app/controllers/users_controller.rb

```diff
@@ -65,6 +63,6 @@ def set_user
 
     # Only allow a list of trusted parameters through.
     def user_params
-      params.require(:user).permit(:name, :email)
+      params.require(:user).permit :name, :email, :password, :password_confirmation
```

Comment:
Khi nào chỉ có 1 đối số thì mới nên bỏ ngoặc em nhé.
```suggestion
      params.require(:user).permit(:name, :email, :password, :password_confirmation)
```